tests/unittests/BackendTestUtils.cpp

```diff
@@ -1131,4 +1131,81 @@ void runOnDevice(ExecutionContext &context, llvm::StringRef name,
   EXIT_ON_ERR(std::move(runErr));
 }
 
+/// Copies a predefined scale and offset into the current row given a pointer
+/// to the scale and offset \p currRowScaleOffsetPtr. Uses \p T as the
+/// datatype for the scale and offset.
+template <typename T>
+static void copyInScaleOffset(char *currRowScaleOffsetPtr) {
+  // Range (0, 255) -> (-0.1, 0.1)
+  T scale = 1.0f / 1275;
+  T offset = -0.1;
+
+  memcpy(currRowScaleOffsetPtr, &scale, sizeof(T));
+  memcpy(currRowScaleOffsetPtr + sizeof(T), &offset, sizeof(T));
+}
+
+Constant *createRandomizedConstant(Module &mod, TypeRef type,
```

Comment:
wanted to put a comment to reuse this in recsys and noticed it's down below :) nice.